Python/pdf_reorder/README.md

```diff
@@ -0,0 +1,32 @@
+
+## A pdf reorder or Rearranging Script
```

Comment:
The README formatting is exceptionally poor. Please take a look at the existing READMEs to have a fair idea of what details you need to add.

Reply:
Okay will do.